rct229/ruleset_functions/get_dict_of_zones_and_terminal_units_served_by_hvac_sys.py

```diff
@@ -0,0 +1,66 @@
+from rct229.utils.jsonpath_utils import find_all
+
+
+def get_dict_of_zones_and_terminal_units_served_by_hvac_sys(rmi):
+    """
+    Returns a dictionary of zones and terminal unit IDs associated with each HVAC system in the RMD.
+
+    Parameters
+    ----------
+    rmi: json
+
+    Returns ------- dict: a dictionary of zones and terminal unit IDs associated with each HVAC system in the RMR,
+    {hvac_system_1.id: {"zone_list": [zone_1.id, zone_2.id, zone_3.id], "terminal_unit_list": [terminal_1.id,
+    terminal_2.id, terminal_3.id]}, hvac_system_2.id: {"zone_list": [zone_4.id, zone_9.id, zone_30.id],
+    "terminal_unit_list": [terminal_10.id, terminal_20.id, terminal_30.id]}}
+    """
+    dict_of_zones_and_terminal_units_served_by_hvac_sys = {}
+    for zone in find_all("$..zones[*]", rmi):
+        zone_id = zone["id"]
+        for terminal in find_all("$.terminals[*]", zone):
+            terminal_id = terminal["id"]
+            hvac_id = terminal.get(
```

Comment:
This is a minor thing. The RDS uses the ```hvac_sys_id``` variable name so, could you check if the variable names need to match?